src/v/config/property.h

```diff
@@ -89,9 +89,10 @@ class property : public base_property {
       : base_property(rhs)
       , _value(std::move(rhs._value))
       , _default(std::move(rhs._default))
-      , _validator(std::move(rhs._validator)) {
-        for (auto binding_ptr : _bindings) {
-            binding_ptr._parent = this;
+      , _validator(std::move(rhs._validator))
+      , _bindings(std::move(rhs._bindings)) {
+        for (auto& binding : _bindings) {
+            binding._parent = this;
```

Comment:
why haven't we seen an issue related to this yet?

Reply:
* property move cases only appear in tests
* since `_bindings` weren't moved over, `this->_bindings` remained empty and the loop was no-op

Reply:
>property move cases only appear in tests

wow, nice find